src/components/Input/HybridDropDown/HybridOption.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PropTypes from "prop-types";
+import DropDownOption from "../DropDown/DropDownOption";
+
+class HybridOption extends React.PureComponent {
+  renderNativeOption = (optionText, children, props) => (
+    <option {...props}> {optionText || children} </option>
+  );
+
+  renderDropDownOption = (children, props) => (
+    <DropDownOption {...props}> {children} </DropDownOption>
+  );
+
+  render() {
+    const {
+      optionFor,
+      optionText,
+      optionProps,
+      dropdownOptionProps,
+      children,
+      ...props
+    } = this.props;
+    if (optionFor === "select") {
+      return this.renderNativeOption(optionText, children, {
+        ...props,
+        ...optionProps
+      });
+    }
+    return this.renderDropDownOption(children, {
+      ...props,
+      ...dropdownOptionProps
+    });
+  }
+}
+HybridOption.propTypes = {
+  value: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+  optionText: PropTypes.string,
+  optionFor: PropTypes.string,
+  /* eslint-disable react/forbid-prop-types */
```

Comment:
We should not disable eslint rule here. We can fix it something like this:

`optionProps: PropTypes.objectOf(PropTypes.object())`

Reply:
It gives an error when i use this.